iceoryx_hoofs/test/moduletests/test_cxx_newtype.cpp

```diff
@@ -30,6 +30,14 @@ struct Sut : public T
 {
     using T::T;
     using T::operator=; // bring all operator= into scope
+
+    // implement ctors and assignment operators when they are implemented by the base class
```

Comment:
You made another moose. 🦌

And as I found out, there is no unicode character for moose... we have to change it